packages/engine/src/ecs/classes/EngineState.ts

```diff
@@ -8,13 +8,16 @@ import { Entity } from './Entity'
 export const EngineState = defineState({
   name: 'EngineState',
   initial: {
```

Comment:
This should be an arrow function not an object, such that Date.now() are called every time EngineState is created, rather than cloning the Date.now() of when the module was loaded